### ISSUE TITLE
Error compiling Less stylesheet

### ISSUE THREAD

**Issue by bjorn-bl** [pycom/pymakr-atom#43]
I get this error message. It seems as there are paths inside the files that belong to Ralphs computer?

Atom is 1.20.1 x64

Error compiling Less stylesheet: C:\Users\Bjørn B. Larsen\.atom\packages\pymakr-atom-1.1.4\styles\pymakr.less
Line number: 7
'../node_modules/font-awesome/less/font-awesome.less' wasn't found. Tried - C:\Users\Bjørn B. Larsen\.atom\packages\pymakr-atom-1.1.4\node_modules\font-awesome\less\font-awesome.less,C:\Users\Bjørn B. Larsen\AppData\Local\atom\app-1.20.1\resources\app\node_modules\one-dark-syntax\node_modules\font-awesome\less\font-awesome.less,C:\Users\Bjørn B. Larsen\AppData\Local\atom\app-1.20.1\resources\app\node_modules\one-dark-ui\node_modules\font-awesome\less\font-awesome.less,C:\Users\Bjørn B. Larsen\AppData\Local\atom\app-1.20.1\resources\app\static\node_modules\font-awesome\less\font-awesome.less,C:\Users\Bjørn B. Larsen\AppData\Local\atom\app-1.20.1\resources\app\node_modules\font-awesome\less\font-awesome.less,..\node_modules\font-awesome\less\font-awesome.less

**Comment by RalphHogenbirk** [comment]
Cleaning up old messages, see we didn't respond to this one. Is this error still happening? Looks like the package folder is a bit off. The 'pymakr-atom-1.1.4' folder should just be 'pymakr'. Did you do a manual install?

**Comment by bjorn-bl** [comment]
Hi

The automatic installation failed, and I tried a manual one. It seems as the original problem with auto install was caused by the fact that my laptop was old. I have switched laptops but not yet installed atom on my new device.

/Bjørn BL.

Bjørn B. Larsen
Førsteamanuensis
NTNU, Inst. for elektroniske system

Fra: Ralph [mailto:notifications@github.com]
Sendt: tirsdag 19. desember 2017 16.07
Til: pycom/pymakr-atom <pymakr-atom@noreply.github.com>
Kopi: Bjørn B. Larsen <bjorn.b.larsen@ntnu.no>; Author <author@noreply.github.com>
Emne: Re: [pycom/pymakr-atom] Error compiling Less stylesheet (#43)


Cleaning up old messages, see we didn't respond to this one. Is this error still happening? Looks like the package folder is a bit off. The 'pymakr-atom-1.1.4' folder should just be 'pymakr'. Did you do a manual install?
